pytorch_lightning/plugins/training_type/tpu_spawn.py

```diff
@@ -121,9 +121,6 @@ def pre_dispatch(self, trainer: "pl.Trainer") -> None:
         if self.debug:
             os.environ["PT_XLA_DEBUG"] = str(1)
 
-        if self.tpu_global_core_rank != 0 and trainer.progress_bar_callback is not None:
```

Comment:
self.global_rank !=0 will do the same thing right?

Reply:
yes. https://github.com/PyTorchLightning/pytorch-lightning/issues/10986